build/automation/lib/file.mk

```diff
@@ -15,7 +15,7 @@ file-replace-content: ### Replace multiline content from given file - mandatory:
 	mv -f $$tmp_file $(FILE)
 	chmod $$permissions $(FILE)
 
-file-replace-variables: ### Replace all placholders in given file - mandatory: FILE; optional: SUFFIX=[variable suffix, defaults to _TO_REPLACE],EXCLUDE_FILE_NAME=true
+file-replace-variables: ### Replace all placholders in given file - mandatory: FILE; optional: SUFFIX=[variable suffix, defaults to _TO_REPLACE],EXCLUDE_FILE_NAME=true,REPLACE_NO_VALUE=true
```

Comment:
Thanks @mabe13 good stuff. Could we have a unit test along with it to cover this use case?

Reply:
@stefaniuk Thanks for checking the code, only recently seen this, just added a unit test. Going to get the `Git Maintainers` group to review it before merging